packages/sdks/src/blocks/text/text.lite.tsx

```diff
@@ -1,8 +1,8 @@
-export default function Text(props: { text: string }) {
+export default function Text(props: { text?: string }) {
   return (
     <span
       class="builder-text"
-      innerHTML={props.text}
+      innerHTML={props.text?.toString()}
```

Comment:
if `innerHtml` always has to be a string this could be changed to
```suggestion
      innerHTML={String(props.text || '')}
```

Reply:
Thanks for the tip! Will change it in my next SDK PR